bin/test

```diff
@@ -27,4 +27,8 @@ lint(){
   cleanGeneratedYamlFiles
 }
 
+deploy(){
+  k8s-deploy-and-verify -f deploy/${PROJECT_NAME}.config
+}
+
 $@
```

Comment:
We don't need this file anymore.

Reply:
I mean, we have separated this file into two: for the backend and the frontend. So we already have `bin/test-frontend.sh` and `bin/test-backend.sh`, we don't need this anymore.

Reply:
Ok let get rid of it in the next PR then.